src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -121,6 +129,43 @@
                 .body(new CaseWorkerProfileCreationResponse("Case Worker Profiles Created."));
     }
 
+    /**
+     * Builds the idam role mappings for case worker roles.
+     * @param serviceRoleMappings list of ServiceRoleMapping
+     * @return list of CaseWorkerIdamRoleAssociation
+     */
+    @Override
+    public IdamRolesMappingResponse buildIdamRoleMappings(List<ServiceRoleMapping> serviceRoleMappings) {
+        List<CaseWorkerIdamRoleAssociation> caseWorkerIdamRoleAssociations = new ArrayList<>();
+        Set<String> serviceCodes = new HashSet<>();
+        IdamRolesMappingResponse idamRolesMappingResponse;
+        serviceRoleMappings.forEach(serviceRoleMapping -> {
```

Comment:
it might fail serviceRoleMappings null. or can't we get null List. If we never get null then it is fine.

Reply:
We will never get null as validation handled in https://github.com/hmcts/rd-caseworker-ref-api/pull/91/files#diff-7366449ef98feb673d5bf328d2b727e040976d2852312488608b41ff82830b0dR124